### DISCUSSION TITLE
2024-03-28 CUE Modules Feedback Session / office hours

### DISCUSSION THREAD

**Discussion by dominikdm** [cue-lang/cue#2990] [Community Calls]
The next Modules Feedback Session will be on **Thursday, Mar 28, 2024 at 1600 UTC.**

See this discussion for context on the modules proposal: https://github.com/cue-lang/cue/discussions/2330

Please reply with any questions/topics you would like to see covered.

### Agenda

* v0.8.0 release update
* Central Registry update - including a demo and details of invites to the alpha version
* Discussion around how to define what is included in a module

### Meeting details

https://meet.google.com/rkk-rbzu-ivk

As usual, this session will be recorded for those unable to join us live.

### CUE Community shared calendar
If you want to stay up to date with future community calls, please [subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).

### Recording

https://youtu.be/vjMC1ghBQp4

### Slides/materials

https://docs.google.com/presentation/d/11LFmpgpe19t5KxxTpr_L-OE3oYH407EuzZKth3ciFgs/edit?usp=sharing

### Auto-generated transcript

To follow.

**Comment by myitcv** [comment]
Updated the description with the agenda for today.

**Comment by myitcv** [comment]
Description updated with links to the recording and slides. Apologies for the delay in posting these!
